project/plugins.sbt

```diff
@@ -3,8 +3,8 @@
 //
 
 // The Lagom plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.3.10")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.0")
 // Needed for importing the project into Eclipse
-addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.0")
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.2.4")
 // The ConductR plugin
-addSbtPlugin("com.lightbend.conductr" % "sbt-conductr" % "2.3.5")
+addSbtPlugin("com.lightbend.conductr" % "sbt-conductr" % "2.5.1")
```

Comment:
won't work with sbt 1.1.0. We need sbt-conductr 2.5.2 with support for sbt 1.1.0